README.md

```diff
@@ -86,3 +86,5 @@ If you need assistance with using Zivid cameras, visit our [**Knowledge Base**](
 
 Zivid Samples are distributed under the [BSD license](https://github.com/zivid/cpp-extra-samples/blob/master/LICENSE).
 
+[ci-badge]: https://img.shields.io/azure-devops/build/zivid-devops/5e76c4a5-26ad-4cbb-8ab5-b9588e1ed2b2/4
```

Comment:
https://img.shields.io/azure-devops/build/zivid-devops/376f5fda-ba80-4d6c-aaaa-cbcd5e0ad6c0/2/master.svg
Is the one we use at: https://github.com/zivid/python-samples
Why do you use this one?

Reply:
Because zivid-python has a different project id than cpp-extra-samples

Reply:
Ok, then we probably have to fix the zivid-samples link shield them. Reused the zivid-python shield there.